Library/Formula/mono.rb

```diff
@@ -39,6 +48,15 @@ def install
     # mono-gdb.py and mono-sgen-gdb.py are meant to be loaded by gdb, not to be
     # run directly, so we move them out of bin
     libexec.install bin/"mono-gdb.py", bin/"mono-sgen-gdb.py"
+
+    # Now build and install fsharp as well
+    resource("fsharp").stage() do
+      ENV.prepend_path "PATH", bin
+      ENV.prepend_path "PKG_CONFIG_PATH", prefix/"lib/pkgconfig"
```

Comment:
Oh sorry I misread your previous comment and I haven't tried that. I can do so tonight.